.github/workflows/scripts/android/commit_and_push.sh

```diff
@@ -0,0 +1,91 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# --------------------------------------------------------------------------------------
+
+#!/bin/bash
+
+$GITHUB_RUN_NUMBER=$1
+$RELEASE_BRANCH=$2
+$MASTER_BRANCH="master"
+
+# Go to root directory
+go_to_root_dir() {
+  cd cd ../../../../
```

Comment:
Is this expected?
```suggestion
  cd ../../../../
```

Reply:
Ack will fix this

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/2/commits/a8896adcad5c91a9c3e873570d395ddd67bf13fe